src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#define dark_theme
```

Comment:
I'm assuming this is not meant to be checked in.

Reply:
it doesn't have to be.  but it makes switching styles nice and easy locally.

Reply:
Can you clarify? I'm not sure how styles work, but switching in VS is easy and relatively fast. I'm not sure we need a separate mechanism.

If we need to code the color for dark theme, this feels like the wrong place/way to do it (since it would be preprocessed out of existence).

---
In reply to: [208730242](https://github.com/dotnet/roslyn/pull/28927#discussion_r208730242) [](ancestors = 208730242)